src/speculators/models/eagle3/config.py

```
@@ -82,3 +82,94 @@ def validate_transformer_config(cls, value: Any) -> PretrainedConfig:
                 ).__class__
             return config_class(**value)
         return value
+# ──────────────────────────────────────────────────────────────────────────────
+# VwnConfig: VWN 参数容器 + 约束校验
+# ──────────────────────────────────────────────────────────────────────────────
+
+@dataclass
```

Ruff (0.15.15)

[error] Undefined name `dataclass`

(F821)